install.md

```diff
@@ -0,0 +1,8 @@
+# Dependenices
```

Comment:
`install.md` should be named `INSTALL.md`

Reply:
have updated the name